package.json

```diff
@@ -61,7 +61,7 @@
     "body-parser": "1.18.3",
```

Comment:
CNR corresponding to this PR has to be included.

Reply:
A note has been added in https://github.com/telefonicaid/iotagent-json/pull/430/commits/5f380c7af75a83c119ccc2810ae015a1909dc087